packages/presentation/lib/screen/details/details_bloc.dart

```diff
@@ -36,6 +45,7 @@ class _DetailsBloc extends BlocImpl<DetailsScreenArguments, DetailsData>
 
   @override
   Future<void> onShareButtonPressed(String message) async {
+    _logAnalyticsEventUseCase('share_button');
```

Comment:
create enum AnalyticsEventType and store events here or at least move event name to const